sorc/ncep_post.fd/EXCH.f

```diff
@@ -72,11 +56,11 @@ SUBROUTINE EXCH(A)
       end
 
 !!@PROCESS NOCHECK
-!
-!--- The 1st line is an inlined compiler directive that turns off -qcheck
-!    during compilation, even if it's specified as a compiler option in the
-!    makefile (Tuccillo, personal communication;  Ferrier, Feb '02).
-!
+!>
+!> @note The 1st line is an inlined compiler directive that turns off -qcheck
+!> during compilation, even if it's specified as a compiler option in the
+!> makefile (Tuccillo, personal communication;  Ferrier, Feb '02).
+!>
```

Comment:
Do we have an author tag for this subroutine?

Reply:
The original code didn't have an author tag on the second subroutine. Should we assume they are from the same author?

Reply:
If you think that's a reasonable assumption, go for it. In cases where I really cannot figure it out I say: 

`@author NOAA Programmer`